src/library/Uno.Material/Styles/Controls/v2/ProgressRing.xaml

```diff
@@ -19,7 +20,7 @@
 	<lottie_not_win:LottieVisualSource x:Key="M3MaterialIndeterminateAnimation_Uno"
 									   UriSource="embedded://Uno.Material/Uno.Material.Assets.MaterialIndeterminate.json" />
 
-	<Style x:Key="MaterialProgressRingStyle"
+	<IsNotWinUI3:Style x:Key="MaterialProgressRingStyle"
```

Comment:
@kazo0 same here please add a comment with more context like you did for the PR description please